config/betterquesting/DefaultQuests.json

```diff
@@ -2124,7 +2124,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Endervoir",
-          "desc:8": "Now that would be nice if you could infinite automated water for your boilers. We got you covered!"
+          "desc:8": "Now wouldn't it be nice if you could have infinite automated water for your boilers? We have got you covered!"
```

Comment:
Looking at it again, I agree that it sounds better without the have in the second sentence. Revert the second sentence (remove "have").